### PR TITLE
Save-in-progress text is not read out all at once

### PR DESCRIPTION
**Pull request by Mottie**
## Summary

- *(Summarize the changes that have been made to the platform)*
  > When the in progress alert is shown on the screen, the text is broken into chunks making a screen reader break what should be a continuous sentence into smaller parts
  >
  > https://user-images.githubusercontent.com/136959/222552162-2c5fd512-9424-40c0-8b56-af1020d17c60.MP4
- *(If bug, how to reproduce)*
- *(What is the solution, why is this the solution)*
  > Instead of adding text with variables inside JSX, we build the text using template strings
- *(Which team do you work for, does your team own the maintenance of this component?)*
  > Benefits decision reviews
- *(If using a flipper, what is the end date of the flipper being required/success criteria being targeted)*

## Related issue(s)

- See [#54518](https://github.com/department-of-veterans-affairs/va.gov-team/issues/54518)
- Related [#1592](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1592)

## Testing done

- *Describe what the old behavior was prior to the change*
  > See the included video
- *Describe the steps required to verify your changes are working as expected*
- *Describe the tests completed and the results*
  > Screenreader testing & updated a failing unit test
- *Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)*

### After

VoiceOver text pane showing the following: heading level 2, Your application is in progress and was last saved on March 2, 2023, at 1:59 p.m. CST is read out all at once
VoiceOver text pane showing the following: You can continue applying now, or come back later to finish your application (Note: extra space before the comma was removed after taking this screenshot)

## What areas of the site does it impact?

All apps that use save-in-progress

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [x]  I added a screenshot of the developed feature
- [x]  [Accessibility foundational testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by rmessina1010 — APPROVED** [review]
LGTM

File: src/platform/forms/save-in-progress/ApplicationStatus.jsx [hunk 1]
```diff
@@ -139,20 +139,21 @@ export class ApplicationStatus extends React.Component {
 
         return (
           <div className="usa-alert usa-alert-info background-color-only sip-application-status vads-u-margin-bottom--2 vads-u-margin-top--0">
-            <h5 className="form-title saved">Your {appType} is in progress</h5>
+            <h5 className="form-title saved">
+              {`Your ${appType} is in progress`}
```
Comment (Andrew565) [inline src/platform/forms/save-in-progress/ApplicationStatus.jsx:143]: Could this instead be `Your {appType} is in progress` without the extra set of brackets and backticks? Seems like this new formatting was applied everywhere you made a change, but I don't see that it's needed.

**Review by Andrew565 — COMMENTED** [review]
Uncertain about formatting changes...

File: src/platform/forms/save-in-progress/ApplicationStatus.jsx (hunk 1 above)
Comment (Mottie) [reply src/platform/forms/save-in-progress/ApplicationStatus.jsx:143]: As noted in our [Slack thread](https://dsva.slack.com/archives/CBU0KDSB1/p1678124939099199?thread_ts=1678121632.471959&cid=CBU0KDSB1), this `ApplicationStatus` component probably isn't used any more if at all... and also Josh was exploring the idea of wrapping content in a `role="text"` which I just tested in the `SaveInProgress` header and it still split the text into chunks... I think I'll just close this issue and we can come back to it once we have a better understanding of the issue and a definite fix.

**Pull request closed without merging**